gits/test/terminal_test.py

```diff
@@ -492,7 +492,7 @@ def _check_cap_cub1(self, pos):
 
         term._cap_cub1()
 
-        if cur_x == 0:
+        if cur_x <= 1:
             self.assertEqual(term._right_most, term._cur_x)
```

Comment:
I didn't get it. Why do we compare `cur_x` with `_right_most` (`_cols - 1`) if `cur_x` is less than or equal to 1?